### ISSUE TITLE
argparse.Namespace usage causes E1101(no-member)

### ISSUE THREAD

**Issue by smoser** [PyCQA/pylint#2413]
### Steps to reproduce
1. get file argparse-namespace.py

       $ cat argparse-namespace.py
       import argparse
       my_ns = argparse.Namespace(my_key='my_value')
       print(my_ns.my_key)

2. run `pylint argparse-namespace.py`

### Current behavior
    $ python3 -m pylint /tmp/argparse-namespace.py
    inside tox:tip-pylint running: python3 -m pylint /tmp/argparse-namespace.py
    ************* Module argparse-namespace
    argparse-namespace.py:4: [E1101(no-member), ] Instance of 'Namespace' has no 'my_key' member

    ------------------------------------
    Your code has been rated at -6.67/10

### Expected behavior
argparse.Namespace is a dynamically created object provided by the standard library.  Its normal usage should not generate errors.

### pylint --version output

    python3 -m pylint --version
    __main__.py 2.1.1
    astroid 2.0.4
    Python 3.6.6 (default, Jun 27 2018, 14:44:17) 
    [GCC 8.1.0]

**Comment by zhangzq** [comment]
I got same error in latest version of pylint. 

```
$ python3 -m pylint --version
__main__.py 2.3.0
astroid 2.1.0
Python 3.6.7 (default, Oct 22 2018, 11:32:17)
[GCC 8.2.0]
```
